webbpsf/wfirst.py

```diff
@@ -285,10 +463,6 @@ class WFI(WFIRSTInstrument):
              simulations, and uses several approximations (e.g. for
              mirror polishing errors, which are taken from HST).
     """
-    # "The H158, F184 and W149 filters and the grism are mounted with proximate cold pupil masks"
-    # from the final draft of the SDT report, page 92, table 3-2
-    UNMASKED_PUPIL_WAVELENGTH_MIN, UNMASKED_PUPIL_WAVELENGTH_MAX = 0.760e-6, 1.454e-6
-    MASKED_PUPIL_WAVELENGTH_MIN, MASKED_PUPIL_WAVELENGTH_MAX = 1.380e-6, 2.000e-6
 
     def __init__(self, set_pupil_mask_on=None):
```

Comment:
I still need it (or something like it), because I need to generate monochromatic PSFs in the overlapping wavelength range (1.38 microns - 1.454 microns) with AND without the pupil mask, so I need a way to explicitly control that.

Reply:
On further inspection and feedback from @rgeda, our code is setting the pupil_mask attribute of the class object, not setting this flag in the init script. So I do get this functionality, in another way.

tl;dr: I actually don't have any need for this code.